session6/src/homework/TernaryEx.java

```diff
@@ -0,0 +1,12 @@
+package homework;
+
+public class TernaryEx {
+    public static void main(String[] args) {
+        int heightOne = 55, heightTwo = 78, maximumHeight;
```

Comment:
I think I mentioned in a comment from previous homework to not use this style of declaring variables, but to declare each variable on different lines. This approach is not recommended since a high number of variables can decrease their visibility and also are harder to maintain - what if you change your mind and want `heightTwo` to be a floating point number? Let's say a double (78.5 instead of 78, for example.) - the line with the entire variables is compromised and shows error.
Also, maximumHeight is not necessary here, but can be directly declared where it is also initialized at line 7:
`int maximumHeight = (heightOne > heightTwo) ? heightOne : heightTwo;`